var/spack/repos/builtin/packages/qhull/package.py

```diff
@@ -47,8 +48,7 @@ class Qhull(Package):
     
     depends_on('cmake')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            cmake('..', *std_cmake_args)
-            make()
-            make("install")
+    @CMakePackage.sanity_check('build')
+    @CMakePackage.on_package_attributes(run_tests=True)
```

Comment:
@citibeth I will have to agree with you there. When CMake crashes, it tells you exactly what went wrong and how to fix it. Autotools just spits out random error messages. I just find CMake packages to be a lot more work to setup. Building in different directories, all of the ugly ALL_CAPS_FLAGS, and each enabled flag adds twenty subflags.